libraries/tock-register-interface/src/lib.rs

```diff
@@ -99,6 +102,8 @@ pub trait UIntLike:
     /// the largest representable value, use a bitwise negation: `~(<T
     /// as UIntLike>::zero())`.
     fn zero() -> Self;
+    fn as_u64(self) -> u64;
+    fn u64_as(v: u64) -> Self;
```

Comment:
The typical naming convention for converting from another type is `fn from_u64(v: u64) -> Self` -- see e.g. the [`num_traits`](https://docs.rs/num-traits/0.2.11/num_traits/cast/trait.FromPrimitive.html) crate.

Reply:
Done.